tool/cmd/kitex/main.go

```diff
@@ -85,4 +86,30 @@ NormalExit:
 		}
 		log.Warn("Adding apache/thrift@v0.13.0 to go.mod for generated code ..........", res)
 	}
+
+	// remove kitex.yaml generated from v0.4.4 which is renamed as kitex_info.yaml
+	if args.ServiceName != "" {
+		DeleteKitexYaml()
+	}
+}
+
+func DeleteKitexYaml() {
+	// try to read kitex.yaml
+	data, err := ioutil.ReadFile("kitex.yaml")
+	log.Info("kitex.y")
```

Comment:
is this for test?

Reply:
Yes， it's a debug output, i've removed it.